app/controllers/concerns/rendering.rb

```diff
@@ -38,5 +48,7 @@ def model_errors(models = [])
     def serializer
       raise NotImplementedError
     end
+
+    def includes; end
```

Comment:
@vkrizan I've added a new feature to this rendering concern which allows controllers to specify any includes to perform prior to serialization.

Reply:
:+1: